mautic/mautic
# ignore unmapped attributes

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [x] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes saml username fallback



#### Description:

SAML without a username mapped does not default to the email when logging in. This causes the user lookup to fail and will attempt to make a new user. Making a new user also fails because the email is already taken.

#### Steps to test this PR:

1. Setup SAML and leave the username config option in CS blank
2. Login as a SAML user (if the user isn't in the account. log out and log back in)
3. You should be able to login
